compiler/rustc_middle/src/ty/fold.rs

```diff
@@ -300,7 +305,8 @@ impl<'tcx> TyCtxt<'tcx> {
         value: &impl TypeFoldable<'tcx>,
         callback: impl FnMut(ty::Region<'tcx>) -> bool,
     ) -> bool {
-        return value.visit_with(&mut RegionVisitor { outer_index: ty::INNERMOST, callback });
+        return value.visit_with(&mut RegionVisitor { outer_index: ty::INNERMOST, callback })
+            == ControlFlow::BREAK;
```

Comment:
pre existing, but can you move that expression to the end of that function and remove the `return`?

This was a bit confusing for me